src/strategies/passport/passport-facebook-oauth2/facebook-auth-strategy-factory-provider.ts

```diff
@@ -0,0 +1,115 @@
+import {inject, Provider} from '@loopback/core';
+import {HttpErrors, Request} from '@loopback/rest';
+import {HttpsProxyAgent} from 'https-proxy-agent';
+import {
+  Profile,
+  Strategy,
+  StrategyOption,
+  StrategyOptionWithRequest,
+} from 'passport-facebook';
+
+import {AuthErrorKeys} from '../../../error-keys';
+import {Strategies} from '../../keys';
+import {VerifyCallback, VerifyFunction} from '../../types';
+
+interface StratOptions extends StrategyOption {
```

Comment:
Interface Name?

Reply:
Does the interface from facebook-passport not have the passReqToCallback property?